api/python/cellxgene_census/src/cellxgene_census/experimental/ml/__init__.py

```diff
@@ -1,6 +1,4 @@
-"""
-An API to facilitate use of PyTorch ML training with data from the CZI Science CELLxGENE Census.
-"""
+"""An API to facilitate use of PyTorch ML training with data from the CZI Science CELLxGENE Census."""
```

Comment:
Inconsistent naming: `CZI Science CELLxGENE Census`

Reply:
FYI this naming is also used in `api/python/cellxgene_census/src/cellxgene_census/__init__.py` and `api/python/cellxgene_census/src/cellxgene_census/experimental/ml/huggingface/__init__.py`